tests/integration/cloud/rest_contracts/test_data_context_configuration.py

```diff
@@ -1,16 +1,30 @@
 from __future__ import annotations
 
 import pathlib
-import uuid
-from typing import Callable
+from typing import Callable, Final
 
 import pytest
-from pact import Like
+from pact import Format, Like
 
 from tests.integration.cloud.rest_contracts.conftest import (
     ContractInteraction,
 )
 
+DATA_CONTEXT_CONFIGURATION_MIN_RESPONSE_BODY: Final[dict] = {
+    "anonymous_usage_statistics": Like(
+        {
+            "data_context_id": Format().uuid,
+            "enabled": False,
```

Comment:
`non-blocking`

Could we use the qualified `pact.Like` and `pact.Format` names?
That way it's more obvious what `Like` and `Format` actually are?
```suggestion
import pact

from tests.integration.cloud.rest_contracts.conftest import (
    ContractInteraction,
)

DATA_CONTEXT_CONFIGURATION_MIN_RESPONSE_BODY: Final[dict] = {
    "anonymous_usage_statistics": pact.Like(
        {
            "data_context_id": pact.Format().uuid,
            "enabled": False,
```

Reply:
Ok I'll do this, but I need to rename a variable and a method that use the work `pact`.